### PR TITLE
Add information for chart repository page

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
###### Description

It is a content for Sumo Logic Helm Repository page, from which the page will be generated.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## Install Sumo Logic Helm Chart
+
+```bash
+helm upgrade --install my-release sumologic/sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
+```
+
+Above command will install the Sumo Logic Helm Chart with the release name `my-release` in the namespace your `kubectl` context is currently set to.
+
+For more details please see [documentation](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/deploy).
```
Comment (sumo-drosiek) [inline index.md:19]: I'm not sure but I think we should point to the main `README.md` or even to the `https://github.com/SumoLogic/sumologic-kubernetes-collection` only

File: _config.yml [hunk 2]
```diff
@@ -1 +1,3 @@
-theme: jekyll-theme-cayman
\ No newline at end of file
+theme: jekyll-theme-cayman
+name: Sumo Logic Helm Chart
+description: A Helm Chart for collecting Kubernetes logs, metrics, traces and events into Sumo Logic
```
Comment (sumo-drosiek) [inline _config.yml:3]: please add new line before EOF

File: index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+```bash
+helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
+```
+
+## Install Sumo Logic Helm Chart
+
+```bash
+helm upgrade --install my-release sumologic/sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
+```
```
Comment (pmalek-sumo) [inline index.md:15]:
```suggestion
```bash
helm upgrade --install my-release sumologic/sumologic \
    --set sumologic.accessId=<SUMO_ACCESS_ID> \
    --set sumologic.accessKey=<SUMO_ACCESS_KEY> \
    --set sumologic.clusterName="<MY_CLUSTER_NAME>"
```
```

Comment (sumo-drosiek) [reply index.md:15]: in addition add `"` to both `<SUMO_ACCESS_KEY>` and `<SUMO_ACCESS_ID>`

Comment (kkujawa-sumo) [reply index.md]: Done:)

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**